spyder_kernels/customize/spydercustomize.py

```diff
@@ -583,6 +571,54 @@ def runfile(filename=None, args=None, wdir=None, namespace=None,
 builtins.runfile = runfile
 
 
+def enter_debugger(filename, continue_if_has_breakpoints, code_format):
```

Comment:
Given that this function is quite long now, I'd prefer to move it to `spyder_kernels/customize/spyderpdb.py`